app/views/authentication_tokens/index.html.erb

```diff
@@ -33,6 +33,15 @@
     <p class="govuk-body">
       You must make sure the token is securely sent to the developers managing your Register API integration.
     </p>
+
+    <form action="<%= new_authentication_token_path %>" method="get" novalidate="">
+      <div class="govuk-button-group">
+        <button type="submit" class="govuk-button" data-module="govuk-button">
+          Generate a new token
+        </button>
+        <a class="govuk-link" href="javascript:history.back();">Cancel</a>
```

Comment:
It's not clear why we use `javascript:history.back();` to go to the organisation settings page and not a `govuk_link_to  "Cancel", organisation_settings_path`